tests/placeholder-search.tests.ts

```diff
@@ -0,0 +1,57 @@
+import { instantMeiliSearch } from '../src'
+import {
+  searchClient,
+  dataset,
+  Movies,
+  meilisearchClient,
+} from './assets/utils'
+
+describe('Pagination browser test', () => {
+  beforeAll(async () => {
+    const deleteTask = await meilisearchClient.deleteIndex('movies')
+    await meilisearchClient.waitForTask(deleteTask.uid)
+    await meilisearchClient
+      .index('movies')
+      .updateFilterableAttributes(['genres'])
+    const documentsTask = await meilisearchClient
+      .index('movies')
+      .addDocuments(dataset)
+    await meilisearchClient.index('movies').waitForTask(documentsTask.uid)
+  })
+
+  test('Test placeholdersearch set to false', async () => {
```

Comment:
What about describing what the test should be expecting?

`Responds with no hits when placeholderSearch is falsy`.